src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -10,11 +10,35 @@
 
 namespace OutGridView.Cmdlet
 {
+    public class ConsoleGuiWindow : Window{
+        public Action Enter_Pressed;
+        public Action Esc_Pressed;
+
+        public ConsoleGuiWindow (string title = null) : base (title) {}
```

Comment:
Please follow the file and repo style. Below too.
```suggestion
        public ConsoleGuiWindow(string title = null) : base(title) {}
```